### PR TITLE
Fix issue 2

### PR DESCRIPTION
**Pull request by GitHubUser5136**

### CODE REVIEW

File: data/Lecturer Course List/222222.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+MAA101 - CALCULUS FOR SCIENCE STUDENTS
+MAT111 - LINEAR ALGEBRA
+MAT181 - PROGRAMMING FOR SCIENTIFIC APPLICATIONS
+MAA111 - ALGEBRA FOR SCIENCE STUDENTS
+LAK100 - KOREAN LANGUAGE
+LAJ100 - JAPANESE LANGUAGE
+LKM400 - BAHASA MALAYSIA
+LSP300 - ACADEMIC ENGLISH
+HFF225 - PHILOSOPHY AND CURRENT ISSUES
+HFE224 - APPRECIATION OF ETHICS AND CIVILISATIONS
```
Comment (GitHubUser5136) [inline data/Lecturer Course List/222222.txt]: Sorry, I forgot to remove all the txt file when git push

File: src/sample/stuDashController.java [hunk 2]
```diff
@@ -203,7 +203,11 @@ public void displayNRIC() {
       Scanner sc = new Scanner(stuinfoFile);
       String icnumber;
       icnumber = Files.readAllLines(Paths.get("data/Student Dashboard/" + id + ".txt")).get(1);
-      nric.setText(icnumber);
+      if(icnumber.contains("empty")){
```
Comment (Michelle-Lohwt) [inline src/sample/stuDashController.java:202]: I think it is no longer  with the word"empty" for the things in this file

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
I think it is no longer with the word"empty" for the things in this file

File: src/sample/stuDashController.java (hunk 2 above)
Comment (GitHubUser5136) [reply src/sample/stuDashController.java:202]: What word is it now?

Comment (Michelle-Lohwt) [reply src/sample/stuDashController.java:202]: nothing I think

Comment (GitHubUser5136) [reply src/sample/stuDashController.java:202]: Gg, my code is outdated, no wonder

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**